flink-libraries/flink-table/src/main/scala/org/apache/flink/api/table/table.scala

```diff
@@ -446,6 +446,31 @@ class Table(
   }
 
   /**
+    * Intersect two [[Table]]s with duplicate records removed. Intersect returns rows only
```

Comment:
The description of intersect suggests that the same row is emitted twice if it appears two times in the left table which is not correct. Rows that appear in both inputs are emitted exactly once.